HuskStandalone/HuskStandalone.param

```diff
@@ -23,4 +23,22 @@ CategoryOrder=0
 Index=0
 Label=HUSK Executable
 Description=The path to the husk executable file used for rendering. Enter alternative paths on separate lines.
-Default=C:\Program Files\Side Effects Software\Houdini 18.5.596\bin\husk.exe
+Default=C:\Program Files\Side Effects Software\Houdini 19.5.000\bin\husk.exe

```

Comment:
Shall we update the default to Hou 20+ since that is currently the latest production build?

Reply:
https://github.com/BigRoy/HuskStandaloneSubmitter/pull/2/commits/cf9b1a247c94cc7dcce0115e80c5d6ef62f8b19b